containers/podman-desktop-extension-bootc
# tests: add podman desktop playwright tests dependency

### What does this PR do?
Introduces the dependency on `@podman-desktop/tests-playwright` e2e testing library for podman desktop.
### Screenshot / video of UI



### What issues does this PR fix or reference?
#198 


### How to test this PR?
```
git clone podman-desktop
cd podman-desktop
yarn install; yarn test:e2e:build
export PODMAN_DESKTOP_ARGS=$(pwd)
cd ..
cd podman-desktop-extension-bootc
yarn install
yarn test:e2e
```
Now the e2e tests from bootc extension should be executed and use podman-desktop that is locally built from codebase.
check `tests/output` folder for the tests results.
